src/native_main.nim

```diff
@@ -105,6 +105,31 @@ proc findUserConfigFile(): string =
         if fileExists(path):
             return path
 
+proc expandVars(path: string): string =
```

Comment:
we should probably wrap this in a

```
when defined posix:
   [this code]
else: 
   return path
```

since environment variables are defined with `%%` on Windows and expanding `$` might cause weird bugs.

(Happy to hear other views if you disagree :))

and thanks for this PR!